### PR TITLE
Updated AL Python tests to use fixturesUtils

### PR DESCRIPTION
**Pull request by J-Mo63**
This PR updates the AL plugin's Python tests to make use of `fixturesUtils`, and removes all uses of `tempfiles` as recommended in https://github.com/Autodesk/maya-usd/pull/1792#discussion_r754488444. Some minor modifications have been made to the utility script for compatibility purposes.

### CODE REVIEW

File: plugin/al/plugin/AL_USDMayaTestPlugin/py/testLayerManager.py [hunk 1]
```diff
@@ -25,26 +25,24 @@
...
-        if os.path.exists(cls._outputPath):
-            shutil.rmtree(cls._outputPath)
-        os.mkdir(cls._outputPath)
-        cls._mayaFilePath = os.path.join(cls._outputPath, 'LayerManagerSerialisation.ma')
-        cls._usdFilePath = os.path.join(cls._outputPath, 'LayerManagerSerialisation.usda')
-
-        # Load the plugin
-        cmds.loadPlugin("AL_USDMayaPlugin", quiet=True)
+        # Load the plugin and setup test output
+        _ = fixturesUtils.setUpClass(__file__, pluginName="AL_USDMayaPlugin")
```
Comment (kxl-adsk) [inline plugin/al/plugin/AL_USDMayaTestPlugin/py/testLayerManager.py:37]: Why assign the return value to `_` if you don't use it? I would prefer either a descriptive name of the variable or simply do not store return value.

File: plugin/al/plugin/AL_USDMayaTestPlugin/py/testProxyShape.py [hunk 2]
```diff
@@ -30,10 +29,21 @@
...
     """Test cases for static function: AL.usdmaya.ProxyShape.getUsdPrimFromMayaPath"""
 
+    @classmethod
+    def setUpClass(cls):
+        # Setup for tests for output
+        _ = fixturesUtils.setUpClass(__file__, loadPlugin=False)
+
+    @classmethod
+    def tearDownClass(cls):
+        _ = fixturesUtils.tearDownClass(unloadPlugin=False)
```
Comment (kxl-adsk) [inline plugin/al/plugin/AL_USDMayaTestPlugin/py/testProxyShape.py:45]: This one is repeated a few times...`tearDownClass` doesn't return any value.

**Review by kxl-adsk — CHANGES_REQUESTED** [review]

File: plugin/al/plugin/AL_USDMayaTestPlugin/py/testLayerManager.py (hunk 1 above)
Comment (J-Mo63) [reply plugin/al/plugin/AL_USDMayaTestPlugin/py/testLayerManager.py:37]: Yeah sorry about that. Bad habit I've imported from Ruby or Lua probably. I'll just remove it.

**Comment by J-Mo63** [comment]
@kxl-adsk I've gone ahead and actioned your feedback.

**Review by kxl-adsk — APPROVED** [review]

**Pull request merged**